test/WebJobs.Script.Tests.Integration/Host/SecretsRepositoryTests.cs

```diff
@@ -614,14 +614,19 @@ public ISecretsRepository GetNewSecretRepository()
                 };
             }
 
-            public void Dispose()
+            public Task InitializeAsync()
+            {
+                return Task.CompletedTask;
+            }
+
+            public async Task DisposeAsync()
             {
                 try
                 {
                     // delete blob files
-                    ClearAllBlobSecrets().ContinueWith(t => { });
+                    await ClearAllBlobSecrets();
                     ClearAllFileSecrets();
-                    ClearAllKeyVaultSecrets().ContinueWith(t => { });
+                    await ClearAllKeyVaultSecrets();
```

Comment:
Bumped into a potential issue where it didn't seem like these secrets were getting cleaned up and some subsequent test failed. Wasn't able to repro but thought I'd make this more deterministic while I was in here.